src/fluree/http_api/handlers/ledger.clj

```diff
@@ -41,14 +41,14 @@
                                     :body   {:error (ex-message t)}}}))))))
 
 (defn txn-body->opts
-  [{:keys [context txn] :as _body}]
+  [{:keys [defaultContext txn] :as _body}]
   (let [first-txn (if (map? txn)
                     txn
                     (first txn))]
     (cond-> {}
             (-> first-txn keys first keyword?) (assoc :context-type :keyword)
             (-> first-txn keys first string?) (assoc :context-type :string)
-            context (assoc :context context))))
+            defaultContext (assoc :defaultContext defaultContext))))
```

Comment:
I think this causes us to ignore `context` on the `/transact` endpoint too, since this fn is called for both. Perhaps the support for `defaultContext` here should be strictly additive?

Reply:
🤔 wait now that I look closer, maybe that's not really a thing. I may be mixed up about the various places context can be supplied

Reply:
Yeah, and/or I might be mixed up too / instead. Let me make sure there is test coverage of that real quick.

Reply:
Actually, looking over the current `fluree.db.json-ld.api/stage` code in db, I don't think you can send a context in the opts of a transaction. Just a context-type. The context would go inside the transaction's JSON-LD data, I believe (i.e. in the map under the `:txn` key).

Reply:
yeah, that sounds right. False alarm! 😅